src/host/srvinit.cpp

```diff
@@ -434,9 +444,9 @@ try
                                                   serverProcess,
                                                   clientProcess.get()));
 
-    inPipeTheirSide.release();
-    outPipeTheirSide.release();
-    signalPipeTheirSide.release();
+    inPipeTheirSide.reset();
+    outPipeTheirSide.reset();
+    signalPipeTheirSide.reset();

```

Comment:
Do you even have to drop those manually when you're exiting the scope a few lines later?

Reply:
I'm just doing it for good measure for the next time someone edits this code and gets the idea to insert more stuff between this code and the bottom.